assets/app/views/browse/route.html

```diff
@@ -0,0 +1,74 @@
+<div ng-controller="RouteController" class="content">
+  <project-page>
+    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+    <alerts alerts="alerts"></alerts>
+    <div ng-if="route">
```

Comment:
Looks like I missed that on all of the individual pages, I'm adding a todo on the trello card to add it for all of them. (will do it in a separate PR)